app/api/entities/reaction_entity.rb

```diff
@@ -18,6 +18,7 @@ class ReactionEntity < ApplicationEntity
       expose! :starting_materials,                                                      using: 'Entities::ReactionMaterialEntity'
       expose! :type
       expose :comment_count
+      expose! :variations, anonymize_with: []
```

Comment:
Du kannst nicht das gleiche Attribut 2x exposen (bzw kannst du schon, dann überschreibt die spätere Definition die frühere (hier gewinnt also die 10er Version in Zeile 49)

Reply:
Weitere Frage: sind die variations strukturstabil oder is das quasi frei für alles? Wenn sie Stabil sind würde ich empfehlen eine eigene Entity dafür zu bauen damit man als API Entwickler sehen kann wie die Struktur aussieht

Reply:
> Du kannst nicht das gleiche Attribut 2x exposen (bzw kannst du schon, dann überschreibt die spätere Definition die frühere (hier gewinnt also die 10er Version in Zeile 49)

Also reicht die erste Definition, richtig?

> Weitere Frage: sind die variations strukturstabil oder is das quasi frei für alles?

Die Variations Datenstruktur is stabil in dem Sinne dass es nicht vorgesehen ist dass Schluessel dynamisch angelegt oder geloescht werden. Ist es das was du mit stabil meinst? Falls ja, lege ich eine Entity an. Hab eh schon nach einem Weg gesucht die Datenstruktur etwas mehr zu formalisieren.

Reply:
> Wenn du sie im below 0 Block lässt werden sie nie anonymisiert. Ist das das was du willst?

Ne, dann hab ich's falsch verstanden, es soll anonymisiert werden. Also lasse ich nur die Definition unter `anonymize_below: 10` stehen 👍

Reply:
@jhmegorei, I added an Entity for the variations, see 7d3ba90